backend/plugin/db/mysql/mysql.go

```diff
@@ -167,10 +167,37 @@ func registerRDSMysqlCerts(ctx context.Context) error {
 
 	rootCertPool := x509.NewCertPool()
 	if ok := rootCertPool.AppendCertsFromPEM(pem); !ok {
-		return err
+		return errors.Errorf("failed to parse RDS CA certificates")
+	}
+
+	// We register both configs upfront for simplicity and performance:
+	// - Registering TLS configs is a one-time operation with minimal overhead
+	// - This avoids conditional logic in getRDSConnection()
+	// - Both configs use the same CA bundle, just different verification settings
+	//
+	// "rds" - maintains backward compatibility (no verification)
+	if err := mysql.RegisterTLSConfig("rds", &tls.Config{
```

Comment:
It might be a bug, not newly introduced though, of using constant tls name "rds". We should use a uuid then.

https://github.com/bytebase/bytebase/blob/c74772d93180952ae28a345e9097e4c9bc5e1220/backend/plugin/db/mysql/mysql.go#L131

tlsKey := uuid.NewString()
if connCfg.DataSource.GetVerifyTlsCertificate() {
  // register InsecureSkipVerify = false
  mysql.RegisterTLSConfig(tlsKey, ...
} else {
  // register InsecureSkipVerify = true
  mysql.RegisterTLSConfig(tlsKey, ...
}
... // later code
return fmt.Sprintf("%s:%s@tcp(%s)/%s?tls=%s&allowCleartextPasswords=true",
		connCfg.DataSource.Username, authenticationToken, dbEndpoint, connCfg.ConnectionContext.DatabaseName, tlsKey,
	), nil